libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/BabbageFeatures.hs

```diff
@@ -121,8 +122,21 @@ collateralOutput :: Scriptic era => Proof era -> Core.TxOut era
 collateralOutput pf =
   newTxOut pf [Address $ plainAddr pf, Amount (inject $ Coin 50)]
 
+-- We intentionally use a ByteString with length greater than 64 to serve as
+-- as reminder that our protection against contiguous data over 64 Bytes on
+-- the wire is done during deserialization using the Plutus library.
+sixtyFiveBytes :: ByteString
+sixtyFiveBytes =
+  "0123456789"
+    <> "0123456789"
+    <> "0123456789"
+    <> "0123456789"
+    <> "0123456789"
+    <> "0123456789"
+    <> "012345"
```

Comment:
It is best to do something like that, because then it is easier to see that there are indeed 65 bytes, to support this claim your implementation has 66 bytes :stuck_out_tongue_winking_eye:
```suggestion
sixtyFiveBytes = BS.pack [1..65]
```

Reply:
yep, that's a great suggestion! I know that haskell does what a mathematician would expect with that notation, it's a closed interval. Other languages can trick you, I know in python `[0, 1][0:1] == [0]`. I've been bitten by this before, but need to remind myself that I'm in a safe place here :laughing:

Reply:
also thanks for sparing me for committing an off by one error while trying to prevent an off by one error :laughing: